src/nih_wayfinding/app/styles/layout/_header-footer.scss

```diff
@@ -42,4 +42,5 @@ footer {
 
 .navbar-actionheader-subtitle {
     padding-right: 11rem;
+    color: #fff;
```

Comment:
Yeah, that grey wasn't working well even on the black. Good call. Separately: this is going to change quite a bit when we implement #111, so I'll be changing the style then.